tests/ui/test_ui_tools.py

```diff
@@ -1026,6 +1026,31 @@ def test_main_view(self, mocker, message, last_message):
         }
         msg_box = MessageBox(message, self.model, last_message)
 
+    @pytest.mark.parametrize('message', [
+        {
+            'id': 4,
+            'type': 'stream',
+            'display_recipient': 'Verona',
+            'stream_id': 5,
+            'subject': 'Test topic',
+            'is_me_message': True,
+            'flags': [],
+            'content': '<p>/me is excited!</p>',
+            'reactions': [],
+            'sender_full_name': 'Alice',
+            'timestamp': 1532103879,
+        }
+    ])
+    @pytest.mark.parametrize('is_me_message', [False, True])
+    def test_main_view_renders_slash_me(self, mocker, message, is_me_message):
+        mocker.patch(VIEWS + ".urwid.Text")
+        self.model.user_full_name = "Foo Boo"
+        message['is_me_message'] = is_me_message
+        msg_box = MessageBox(message, self.model, message)
+        msg_box.main_view()
+        assert (msg_box.message['content'].find(
+            self.model.user_full_name) >= 0) is is_me_message
```

Comment:
Would it be better to use `==`instead of `is`?

Reply:
yeah, an explicit approach looks better.